app/src/androidTest/java/com/android/voyageur/ui/search/SearchUserProfile.kt

```diff
@@ -0,0 +1,104 @@
+package com.android.voyageur.ui.search
+
+import androidx.compose.ui.test.assertIsDisplayed
```

Comment:
Moreover, your test coverage is too low, especially for the newly added SearchUserProfile, you should aim for >=80% line coverage for the M2.

Reply:
I've added tests that increase coverage, specifically targeting key functionality within SearchUserProfile. While the tests cover most of the essential interactions and logic, I aimed to focus on the areas that are critical to user experience and screen stability.